docs/autogen/apigen.py

```diff
@@ -3,44 +3,48 @@
     Parse the xNVMe header files for auto-generation purposes
 """
 from __future__ import print_function
-from subprocess import Popen, PIPE
+
 import argparse
-import logging
-import pprint
 import copy
-import sys
+import logging
 import os
+import sys
+from subprocess import PIPE, Popen
+
 import jinja2
 
 DECLARATIONS = {
     "func": [],
     "macro": [],
     "struct": [],
     "enum": [],
-}
+}  # type: dict
```

Comment:
Is it `mypy` that requires this? Why not:
```python
DECLARATIONS : dict = {
     "func": [],
     "macro": [],
     "struct": [],
     "enum": [],
 }
```

Reply:
Yup, mypy complains when the type-annotation is not there